canonical/ubuntu.com
# WD-10664 - Fix the broken links on desktop/wsl

## Done

- Removed the duplicated root level wsl page in favour of the subdirectory one
- Updated the link to the contact page
- Removed the unnecessary modals.js include

## QA

- Open the demo
- Go to /desktop/wsl
- See that when you middle click the "Looking for support? Contact us ›" link it opens an active form page
- Check the console have no errors loading js

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-10664
Fixes https://github.com/canonical/ubuntu.com/actions/runs/8796442342/job/24139308386
